### PR TITLE
Updated Line 78, Line 91

### PR DESCRIPTION
**Pull request by myratsapar**
Line 78  - added zero (0) next to return. Function type is void so there will be no return value. 
Line 91 - added parentheses  second part of if condition. Purpose of two renewed lines are just increase readability for client. Peace Guys!

### CODE REVIEW

File: libraries/EEPROM/EEPROM.cpp [hunk 1]
```diff
@@ -75,7 +75,7 @@ void EEPROMClass::begin(size_t size) {
 
 void EEPROMClass::end() {
   if (!_size)
-    return;
+    return 0; //Added zero to increase readability for clients
```
Comment (devyte) [inline libraries/EEPROM/EEPROM.cpp]: Nope, return type is void, so no return value.

File: libraries/EEPROM/EEPROM.cpp [hunk 2]
```diff
@@ -88,7 +88,7 @@ void EEPROMClass::end() {
 
 
 uint8_t EEPROMClass::read(int const address) {
-  if (address < 0 || (size_t)address >= _size)
+  if (address < 0 || ((size_t)address >= _size)) //Added brackets second part of condition for more readability
```
Comment (devyte) [inline libraries/EEPROM/EEPROM.cpp]: If you know the operator precedence, then it's obvious. However, i don't mind the parentheses. Please drop the comment though, it explains your intent to us, but it's not helpful for the code.

Comment (myratsapar) [reply libraries/EEPROM/EEPROM.cpp]: I do appreciate your respond, devyte. I didn't even think about that you will notice my little correction request on large project like this.  This is my first pull-request operation. I'm suppose to do pull-request job for my homework in open source project. I really interested Arduino and its modules. That i could be part of this project. I don't know culture of github yet. But i'll get used to it.

File: libraries/EEPROM/EEPROM.cpp (hunk 1 above)
Comment (myratsapar) [reply libraries/EEPROM/EEPROM.cpp]: I am totally agree with u. I don't know what i thought last night when i typed this. Now i see for my code's condition is different.

**Comment by devyte** [comment]
BTW, you're making the pr from your fork's master branch. You should create a new branch and push from there. Then, once the pr is merged the branch should be deleted.

**Comment by devyte** [comment]
Closing due to lack of response in over a month.

**Pull request closed without merging**